hooks4git/scripts.py

```diff
@@ -102,7 +102,15 @@ def add_hooks(path=os.environ["PWD"]):
         if git_path:
             path = os.path.abspath(path)
             setup_path = os.path.abspath(setup_path)
-            if os.path.isdir(os.path.join(git_path, "hooks")):
+            hooks_path = os.path.join(git_path, "hooks")
```

Comment:
extract this to a method, _e.g._ `ensure_hooks_dir_exists`, so you can easily test it by giving it a fake directory

Reply:
I didn't see it, sorry. I was following up on the issue. I will work on this.